ddtrace/tracer/context.go

```diff
@@ -35,9 +35,11 @@ func SpanFromContext(ctx context.Context) (Span, bool) {
 	return &internal.NoopSpan{}, false
 }
 
-// StartSpanFromContext returns a new span with the given operation name and options. If a span
-// is found in the context, it will be used as the parent of the resulting span. If the ChildOf
-// option is passed, the span from context will take precedence over it as the parent span.
+// StartSpanFromContext returns a new span with the given operation name and
+// options. If a span is found in the context, it will be used as the parent of
+// the resulting span. If the ChildOf option is passed, the span from context
+// will take precedence over it as the parent span. The WithContext is also
+// ignored.
```

Comment:
```suggestion
// the resulting span. If the ChildOf option is passed, the span from ctx
// will take precedence over it as the parent span. The WithContext is also
// ignored.
```
Maybe a better way to explain is to list the precedence:
```go
// The order of precedence for considering the parent span when multiple options are set
// simultaneously is:
//  1. Context function argument (ctx)
//  2. WithContext StartSpanOption
//  3. ChildOf StartSpanOption
```
I'm ok with either. Just a suggestion.